h/static/styles/partials-v2/_search.scss

```diff
@@ -230,13 +230,6 @@
 // On normal tablets and below, display annotation stats below annotation list
 @media screen and (max-width: $tablet-width) {
 
-  // Reduce margins on tablet and below to match navbar left/right margins
-  // on these screen sizes
-  .search-result-list {
-    margin-left: 10px;
-    margin-right: 10px;
-  }
```

Comment:
The purpose of this was to _reduce_ the margins of the search result list on small (phone) sized devices. The result should be that the left and right edges of annotation cards are horizontally aligned with the navbar at the top.

Failing to do this wastes horizontal space on small screens.